mod-source-record-storage-server/src/main/java/org/folio/services/entities/ConsortiumConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package org.folio.services.entities;
+
+public class ConsortiumConfiguration {
```

Comment:
@RomanChernetskyi  Let's add some documentation for the new classes.

Reply:
Thanks, done